### DISCUSSION TITLE
Does joyride have support for react v17

### DISCUSSION THREAD

**Discussion by ghost** [gilbarbara/react-joyride#939] [General]
Hello, everyone! I'm currently working on a React project using version 17. My question is: does React Joyride support React 17? If it does, which version of React Joyride should I install? Any help would be greatly appreciated.

**Comment by gilbarbara** [comment]
Hey @pathu-12 

Yes, v2 still supports [react 17.x](https://github.com/gilbarbara/react-joyride/blob/main/package.json#L33)
